### DISCUSSION TITLE
SyntaxError: invalid syntax

### DISCUSSION THREAD

**Discussion by standyyyy** [openai/whisper#945] [Q&A] [answered]
The python3.7 and 3.9 environments have been tried, but errors were reported when running
Traceback (most recent call last):
  File "F:/Aawant/jinzhuan/audio.py", line 8, in <module>
    import whisper
  File "D:\Anaconda3\lib\site-packages\whisper\__init__.py", line 12, in <module>
    from .decoding import DecodingOptions, DecodingResult, decode, detect_language
  File "D:\Anaconda3\lib\site-packages\whisper\decoding.py", line 520
    if prefix := self.options.prefix:
               ^
SyntaxError: invalid syntax

After comparing the decoding files on github, they found the same result. Now I don't know what to do

**Comment by jongwook** [comment, marked as answer]
It should run on Python 3.9 without `SyntaxError`. If the problem persists, you could also try the latest version that worked with Python 3.7 with:

```
pip uninstall whisper
pip install --force-reinstall openai-whisper==20230124
```

**Reply by KanrongYu** [reply]
works for me, thanks! and to save 1 min editing time, it should be 
```
pip uninstall openai-whisper
pip install --force-reinstall openai-whisper==20230124
```
